src/components/ErrorMessage/ErrorMessage.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import cN from 'classnames';
+
+type Props = {
+  errorText: null | string,
```

Comment:
The `errorText` prop can be marked as optional, because a `null` value can be passed. 
Update the type to `errorText?: string` for cleaner code.